workers/unity/Packages/com.improbable.gdk.transformsynchronization/Systems/InterpolateTransformSystem.cs

```diff
@@ -32,8 +32,7 @@ protected override void OnCreateManager(int capacity)
         {
             base.OnCreateManager(capacity);
 
-            var worker = Core.Worker.GetWorkerFromWorld(World);
-            origin = worker.Origin;
+            origin = World.GetExistingManager<WorkerSystem>().Origin;
```

Comment:
this will simply fail with a null pointer exception, I believe. do we not want to provide a better exception for this?

Reply:
Can do. Can have a convention for feature modules that we check for there being a worker.

Reply:
I don't think it's worth going full error handling in transforms right now though as that module is going to just be re-written.

Reply:
that was more a remark for all systems, just happened to comment on it in the transform system

Reply:
Sure. But I don't think that is part of this PR.